src/components/FeaturePanel/EditDialog/useGetOnClose.tsx

```diff
@@ -10,7 +10,8 @@ export const useGetOnClose = () => {
   return () => {
     close();
     if (successInfo?.redirect) {
-      router.replace(successInfo.redirect); // only useRouter reloads the panel client-side
+      // Reloads the panel client-side
+      router.replace('/').then(() => router.replace(successInfo?.redirect));
```

Comment:
Could you try redirecting user to url like `/node/2432753840?version=7`? 

I think we may be able to get the version from the osmApi. And it wouldn't run an unneccesary request to `/` in between, thus making it quicker.

btw, i will be vacationing for next few weeks, mostly offline. 🏖️😃 @Dlurak  can help you merge this if he has time

Reply:
Tried router.replace with:
- /node/597644745 to /node/597644745/4 ❌ not working
- /node/597644745 to /node/597644745?v=4 ❌ not working

Reply:
Thanks for the check. And thanks @dlurak for merging. 👌